CONTRIBUTING.rst

```diff
@@ -0,0 +1,18 @@
+Contributing to pipenv
+======================
+
+To work on pipenv itself, fork the repository and clone your fork to your local
+system.
+
+Now, install the development requirements::
+
+    cd pipenv
+    virtualenv ~/pipenv-venv  # You can use a different path if you like.
+    source ~/pipenv-venv/bin/activate
+    pip install -e . --upgrade
```

Comment:
I would probably suggest using `python setup.py develop` here since it's more idiomatic.

Reply:
I was trying to stay close to what we do on travis, where we use ` pip install -e . --upgrade`.

Do you prefer to just use `setup.py develop here`, or also make the same change to `.travis.yml`?

Reply:
setup.py develop is just a more universal way of installing a package locally for development and has historically been how Kenneth and I ran our dev environments.

It looks like the .travis.yml file was implemented with this syntax when we reintroduced Travis a couple months ago. @erinxocon, any particular reason we should use that syntax?

Reply:
I changed it to `python setup.py develop` for now. I guess it's easy enough to change it later if you want to.